### PR TITLE
8.2 Install and setup automapper, 8.5 Move repository interface to business layer

### PR DESCRIPTION
**Pull request by sub13**
I added and i set automapper up, i little refactored code, i fixed two bug in our program: 1) bug in getSprintById view; 2) bug in alltasksforteamView. If i've done something wrong please write down. I'll try to fix as soon as i can.
UPDATE1:
I Added Lazy loading into our project, I replaced all repositories to generic one repository, I removed business word, I added Imodel interface with method Update. And i set up tests for these changes

### CODE REVIEW

**Review by SergeyMokin — CHANGES_REQUESTED** [review]
Required updates from call

**Comment by sub13** [comment]
UPDATE:
I Added Lazy loading into our project, I replaced all repositories to generic one repository, I removed business word, I added Imodel interface with method Update. And i set up tests for these changes

File: src/Teams.Data/Repository/IRepository.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Teams.Data.Repository
```
Comment (SergeyMokin) [inline src/Teams.Data/Repository/IRepository.cs:4]: Why it is not in Bussiness layer?

**Review by SergeyMokin — CHANGES_REQUESTED** [review]
Move project dependencies back

**Review by SergeyMokin — APPROVED** [review]

**Pull request merged**